docs/e2e/running.md

```diff
@@ -0,0 +1,59 @@
+#### Running e2e tests
+
+To run the e2e tests locally it is recommended to use the [e2e](https://quay.io/repository/openshift-on-azure/e2e-tests)
+container image
+
+```
+docker pull quay.io/openshift-on-azure/e2e-tests:latest
+```
+
+Example: Run the end user e2e tests
+```
+export FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]\[Fake\]"
+export TIMEOUT=20m
+
+docker run \
+--rm \
+-it \
+--volume $PWD/_data:/_data \
+--volume $PWD/artifacts:/artifacts \
+--volume $PWD/secrets:/secrets \
+--volume $HOME/.kube/config:/root/.kube/config \
+quay.io/openshift-on-azure/e2e-tests \
+-test.v \
+-test.timeout=${TIMEOUT} \
+-ginkgo.noColor \
+-ginkgo.v \
+-ginkgo.focus=${FOCUS:-} \
+-artifact-dir=/artifacts
+```
+
+Example: Run the scale up/down e2e tests against the fake rp
+```
+export FOCUS="\[ScaleUpDown\]\[Fake\]"
+export TIMEOUT=30m
+source ./env
+
+docker run \
+--rm \
+-it \
+--env AZURE_CLIENT_ID=${AZURE_CLIENT_ID} \
+--env AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET} \
+--env AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID} \
+--env AZURE_TENANT_ID=${AZURE_TENANT_ID} \
+--env RESOURCEGROUP=${RESOURCEGROUP} \
+--volume $PWD/_data:/_data \
+--volume $PWD/artifacts:/artifacts \
+--volume $PWD/secrets:/secrets \
+--volume $PWD/test/manifests:/manifests \
+--volume $HOME/.kube/config:/root/.kube/config \
+quay.io/openshift-on-azure/e2e-tests \
+-test.v \
+-test.timeout=${TIMEOUT} \
+-ginkgo.noColor \
+-ginkgo.v \
+-ginkgo.focus=${FOCUS:-} \
+-artifact-dir=/artifacts \
+-scaleUpManifest= /manifests/normal/scaleup.yaml \
```

Comment:
Is the extra space between the `=` sign not breaking the command? For completeness, I would remove the space.